images/dns-controller/Dockerfile

```diff
@@ -21,4 +21,6 @@ RUN apt-get update && apt-get install --yes ca-certificates \
 
 COPY /.build/artifacts/dns-controller /usr/bin/dns-controller
 
+USER 1000
```

Comment:
For my own understanding, do we need to also set the user in the `securityContext` of the `dns-controller` here: https://github.com/kubernetes/kops/blob/master/upup/pkg/fi/cloudup/tests/bootstrapchannelbuilder/simple/dns-controller.addons.k8s.io-k8s-1.12.yaml

Reply:
We will, but we need someone to release new images built from this `Dockerfile` first. So that's a different PR.